src/app/admin/modules/data-management/components/visual-graph-param-selector/visual-graph-param-selector.component.html

```diff
@@ -0,0 +1,68 @@
+<div class="container-fluid">
+  <div class="row" [formGroup]="selectedGraph">
+      <div class="col-md-2">
+        <mat-form-field>
+          <mat-label>选择统计类型</mat-label>
+          <mat-select formControlName="selectedStatisticsType">
+            <mat-option *ngFor="let statistics of statisticsType" [value]="statistics.type">{{statistics.name}}</mat-option>
+          </mat-select>
+        </mat-form-field>
+      </div>
+
+      <div class="col-md-2" *ngIf="selectedGraph.get('selectedStatisticsType').value !== null">
+        <mat-form-field>
+          <mat-label>选择分组类型</mat-label>
+          <mat-select formControlName="selectedGroupType">
+            <mat-option *ngFor="let groupType of secondType[selectedGraph.get('selectedStatisticsType').value]" [value]="groupType.type">{{groupType.name}}</mat-option>
+          </mat-select>
+        </mat-form-field>
+      </div>
+      <div class="col-md-2">
+        <mat-form-field>
+          <mat-label>输入起始年份</mat-label>
+          <mat-select  formControlName="selectedStartYear" required>
+            <mat-option>--</mat-option>
+            <mat-option *ngFor="let year of yearList" [value]="year">{{year}}</mat-option>
+          </mat-select>
+          <mat-error *ngIf="selectedGraph.get('selectedStartYear').hasError('required')">Please choose a right start year</mat-error>
+        </mat-form-field>
+        <div *ngIf="selectedGraph.errors?.timeValidator" class="cross-validation-error-message alert alert-danger">请选择正确的开始年份</div>
+      </div>
+      <div class="col-md-2">
+        <mat-form-field>
+          <mat-label>输入终止年份</mat-label>
+          <mat-select formControlName="selectedEndYear" required>
+            <mat-option>--</mat-option>
+            <mat-option *ngFor="let year of yearList" [value]="year">{{year}}</mat-option>
+          </mat-select>
+          <mat-error *ngIf="selectedGraph.get('selectedEndYear').hasError('required')">Please choose a right end year</mat-error>
+        </mat-form-field>
+        <div *ngIf="selectedGraph.errors?.timeValidator" class="cross-validation-error-message alert alert-danger">请选择正确的结束年份</div>
+      </div>
+      <div class="col-md-2" *ngIf="showDepartmentSelector">
+        <mat-form-field>
+          <mat-label>选择学院</mat-label>
+          <mat-select formControlName="selectedDepartment">
+            <mat-option *ngFor="let department of departmentsList" [value]="department">{{department}}</mat-option>
+          </mat-select>
+        </mat-form-field>
+      </div>
+  </div>
+  <div *ngIf="selectedGraphValues">
+  <div class="row">
```

Comment:
When possible, please use [flex-layout](https://github.com/angular/flex-layout/wiki/API-Documentation) instead of grid-layout, and double-check the appearance of your HTML in mobile platform, we need to build responsive HTML for our users.